ferrilata-devilline.IntegrationTests/Scenarios/ApiControllers/AdminController/ApiAdminAddTests.cs

```diff
@@ -0,0 +1,136 @@
+using Newtonsoft.Json;
+using System.Collections.Generic;
+using System.Net;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+using ferrilata_devilline.IntegrationTests.Fixtures;
+using ferrilata_devilline.Models.DTOs;
+
+namespace ferrilata_devilline.IntegrationTests.Scenarios
+{
+    [Collection("BaseCollection")]
+    public class ApiAdminAddTests
+    {
+        private readonly TestContext _testContext;
+        private readonly HttpRequestMessage _message;
+
+        public ApiAdminAddTests(TestContext testContext)
+        {
+            _testContext = testContext;
+            _message = new HttpRequestMessage(HttpMethod.Post, "/api/admin/add");
+        }
+
+        #region test body setup
+        public static IEnumerable<object[]> MissingFields =>
+            new List<object[]>()
+            {
+                new object[] { new StringContent(JsonConvert.SerializeObject(
+                    new { version = "2.3", levels = new List<object>() }), 
+                    Encoding.UTF8, "application/json") },
+            };
+
+        public static IEnumerable<object[]> Correct =>
+            new List<object[]>()
+            {
+                new object[] { new StringContent(JsonConvert.SerializeObject(
+                    new { version = "2.3", name = "Badge inserter", tag = "general", levels = new List<object>() }),
+                    Encoding.UTF8, "application/json") },
+            };
+
+        public static IEnumerable<object[]> NullValue =>
+            new List<object[]>()
+            {
+                new object[] { new StringContent(JsonConvert.SerializeObject(
+                    new AdminDTO { Version = null, Name = "Badge inserter", Tag = "general", Levels = new List<object>() }), 
+                    Encoding.UTF8, "application/json")},
+            };
+        #endregion
+
+        [Theory]
+        [MemberData(nameof(Correct))]
+        public async Task Authorized_AndHasCorrectBody_Created(HttpContent content)
+        {
+            _message.Headers.Add("Authorization", "something");
+            _message.Content = content;
+
+            var response = await _testContext.Client.SendAsync(_message);
+
+            Assert.Equal(HttpStatusCode.Created, response.StatusCode);
+        }
+
+        [Theory]
+        [MemberData(nameof(Correct))]
+        public async Task Authorized_AndHasCorrectBody_ResponseObject(HttpContent content)
+        {
+            var expectedResponseObject = new List<object>() { new { message = "Created" } };
+            string expectedResponseString = JsonConvert.SerializeObject(expectedResponseObject);
```

Comment:
It is easiert to just simply give the expected string as it is, instead of converting it from some object using more lines